src/data/array_types.js

```diff
@@ -941,6 +1005,96 @@ export class PlacedSymbolArray extends StructArrayLayout2i2ui3ul3ui2f2ub40 {
 
 register('PlacedSymbolArray', PlacedSymbolArray);
 
+class SymbolInstanceStruct extends Struct {
```

Comment:
running `yarn run codegen` gets rid of this. Do you have some uncommitted changes?

Reply:
😳 OMG you mean I didn't have to make that all by hand. 🤦‍♂️